srcpkgs/pdf.js/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdf.js'
+pkgname=pdf.js
+version=1.10.100
+revision=1
+noarch=yes
+hostmakedepends="nodejs"
+short_desc="Portable Document Format (PDF) viewer built with HTML5"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="Apache-2.0"
+homepage="https://mozilla.github.io/pdf.js/"
+distfiles="https://github.com/mozilla/pdf.js/archive/${version}.tar.gz"
+checksum=6f55b1eb85e8912296c6540e8b2e09a96f832caf9a0eba41a7a0acc41ec74cf6
+
+do_build() {
+	npm install -g gulp-cli
```

Comment:
Hm, can't you install this locally and simply access the binaries that is in `.node_modules`? Otherwise this will persist between builds because it won't be removed by xbps-src automatic removing of auto{,cross}deps.

Reply:
Fixed!